README.md

```diff
@@ -27,13 +46,13 @@ export default class MyComponent extends React.Component {
     return (
       <React.Fragment>
         <ComponentToPrint ref={this.componentRef} />
-        <button onClick={() => exportComponentAsJPEG(this.componentRef)}>
+        <button onClick={() => exportComponentAsJPEG({node: this.componentRef})}>
```

Comment:
`node`  should still be passed directly otherwise it will break existing usage.

Reply:
i addressed this in the latest commit